firmware/application/ui_navigation.cpp

```diff
@@ -178,7 +178,7 @@ const NavigationView::AppList NavigationView::appList = {
     {"aprstx", "APRS TX", TX, ui::Color::green(), &bitmap_icon_aprs, new ViewFactory<APRSTXView>()},
     {"bht", "BHT Xy/EP", TX, ui::Color::green(), &bitmap_icon_bht, new ViewFactory<BHTView>()},
     {"bletx", "BLE Tx", TX, ui::Color::green(), &bitmap_icon_btle, new ViewFactory<BLETxView>()},
-    //{"keyfob", "Key fob", TX, ui::Color::orange(), &bitmap_icon_keyfob, new ViewFactory<KeyfobView>()},
+    //{"keyfob", "Key fob", TX, ui::Color::orange(), &bitmap_icon_keyfob, new ViewFactory<KeyfobView>()}, //moved into ext
```

Comment:
I was tempted to remove all the commented out apps, but then Dr. SDR came along just at the right time trying to reenable acars for me to keep them. Still removed external apps however, they no longer belong here IMO :)

Moving to external comes with a whole bunch of changes, if we want to re-internalize them I'd advise trying a reverse commint instead of removing comments..

Reply:
sure thing, will fix now

Reply:
thx, fixed